ir_filters_multi_company/views/ir_filters.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2019 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <record model="ir.ui.view" id="view_ir_filters_form">
+        <field name="model">ir.filters</field>
+        <field name="inherit_id" ref="base.ir_filters_view_form" />
+        <field name="arch" type="xml">
+            <field name="user_id" position="after">
+                <field name="company_id" groups="base.group_multi_company" />
+            </field>
+        </field>
+    </record>
+
+
+    <record model="ir.ui.view" id="view_ir_filters_tree">
+        <field name="model">ir.filters</field>
+        <field name="inherit_id" ref="base.ir_filters_view_tree" />
+        <field name="arch" type="xml">
+            <tree position="inside">
+                <field name="company_id" groups="base.group_multi_company" />
+            </tree>
+        </field>
+    </record>
+
```

Comment:
could you also add the field in the search view ? base.ir_filters_view_search
thanks !